.github/workflows/sbom.yaml

```diff
@@ -13,19 +13,23 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Download Syft
-        id: syft
-        uses: anchore/sbom-action/download-syft@v0.14.2
-
       - name: Generate SBOM from source code
-        run: "${{ steps.syft.outputs.cmd }} --config .github/edgebit/source-syft.yaml --file /tmp/sbom.syft.json ."
+        uses: eyakubovich/sbom-action@ey/add-config-input
+        with:
+          artifact-name: sbom.spdx.json
+          upload-artifact: true
+          config: .github/edgebit/build-syft.yaml
```

Comment:
should be `source-syft.yaml` -- not sure why it didn't error out

Reply:
Also remove `output: syft` from that config file. That tells it to generate a syft (not SPDX) SBOM. In this case SPDX is fine (and the action will request that).

Reply:
Good catch.